src/app/Application.cpp

```diff
@@ -192,3 +198,46 @@ void Application::updateReady(QString updaterPath) {
     QMessageBox::critical(mainWindow, "Update failed!", "Couldn't update. Please try to update manually.");
 }
 
+void Application::noImagesToDisplay() {
```

Comment:
The application is getting quite chonky, should probably start moving these messageboxes out to a different class in the future.